cloud/cluster/memory/node.go

```diff
@@ -0,0 +1,48 @@
+package memory
+
+import (
+	"fmt"
+	"github.com/scootdev/scoot/cloud/cluster"
+)
+
+type idNode struct {
+	id cluster.NodeId
+}
+
+func (n *idNode) Id() cluster.NodeId {
+	return n.id
+}
+
+func (n *idNode) String() string {
+	return string(n.id)
+}
+
+func NewIdNode(id string) cluster.Node {
+	return &idNode{cluster.NodeId(id)}
+}
+
+var _ cluster.Node = (*idNode)(nil)
+
+func NewIdNodes(num int) []cluster.Node {
```

Comment:
I think a valid impl would allow you to to create a set of nodes that start at id>0. Otherwise, this seems hardcoded and therefore 'fake' (and that's my understanding of fake, you've hardcoded behavior that would otherwise be more dynamic).